cicecore/cicedynB

```diff
@@ -0,0 +1 @@
+cicedyn
```

Comment:
I'm OK with adding a symlink for now, but maybe we could add a note in the release notes that this is temporary and will be removed in a future version ?